boards/FSM/Src/main.c

```diff
@@ -44,6 +44,7 @@
 #include "App_FsmWorld.h"
 #include "App_InRangeCheck.h"
 #include "App_SharedStateMachine.h"
+#include "App_SharedBinaryStatus.h"
```

Comment:
```suggestion
```

Reply:
This should have been included in `App_FsmWorld.h` so there's no need to do it here

Reply:
fixed